ecommerce/docs/api/business-policies.md

```
@@ -0,0 +1,308 @@
+# 이커머스 비즈니스 정책 정의서
+
+## 필수 비즈니스 정책
+
+### 1. 장바구니 관리 정책
+
+#### 1.1 최대 장바구니 항목 제한
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)